app/models/dialog_field_text_box.rb

```diff
@@ -25,6 +25,7 @@ def value_from_dialog_fields(dialog_values)
   end
 
   def automate_output_value
+    return nil if @value.nil?
```

Comment:
@gmcculloug It used to work when the value from `automate_output` is `nil`. The problem was introduced during our recent work in #8401